doc/nrf/protocols/wifi/sap_mode/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _ug_wifi_softap_mode:
+
+SoftAP mode
+###########
+
+The following subpages describe the memory requirements of the SoftAP sample and how an nRF70 Series device operates in :term:`Software-enabled Access Point (SoftAP or SAP)` mode.
```

Comment:
I'd say let's make this text more generics, as we are likely going to add more examples with SoftAP mode.